### PR TITLE
Smol fix for grub event.

### PR DESCRIPTION
**Pull request by Ketrai**
This makes it so grubs are less tedious to deal with. They always spawn with around 4, and always in the same area. This way security shouldn't have to hunt them down.

- Also for people who don't know yet, their chem injection has been fixed, so now they don't do insane amounts of damage like they're not supposed to anyways.

### CODE REVIEW

**Comment by Ketrai** [comment]
How is there a dme error when I literally downloaded a fresh branch like an hour ago?

File: code/modules/events/grubinfestation_vr.dm [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
+			var/spawn_type = pick(spawn_types)
+			new spawn_type(T)
 
-	while((spawncount >= 1) && vents.len)
-		var/obj/vent = pick(vents)
-		new /obj/effect/spider/spiderling/grub(vent.loc)
-		vents -= vent
-		spawncount--
+/datum/event/grub_infestation/announce()
+	command_announcement.Announce("Bioscans indicate that solar grubs have somehow entered through the gateway. Clear them out, before this starts to affect the powergrid.", "Vermin infestation")
```
Comment (SpadesNeil) [inline code/modules/events/grubinfestation_vr.dm]: The gateway? Do they only ever spawn in the gateway?

Comment (Ketrai) [reply code/modules/events/grubinfestation_vr.dm]: Yup, any other areas you want them to spawn in? people complained about them ending up all over the station, even in like telecoms

Comment (SpadesNeil) [reply code/modules/events/grubinfestation_vr.dm]: I say let them show up anywhere except some blacklisted areas, but _not_ use vents.

Or conversely, whitelisted areas, as defined in the original infestation code for mice and lizards.

Comment (Ketrai) [reply code/modules/events/grubinfestation_vr.dm]: Actually am going to gather opinions on this via a vote.

Comment (Ketrai) [reply code/modules/events/grubinfestation_vr.dm]: https://strawpoll.com/5c8ry1s7

Comment (SpadesNeil) [reply code/modules/events/grubinfestation_vr.dm]: "All over station, excluding the dorms" has huge majority compared to other stuff.

Comment (Ketrai) [reply code/modules/events/grubinfestation_vr.dm]: Alrite, i'll close this pull for now and go work on a fix so they can still spawn everywhere, just not the dorms.

**Pull request closed without merging**